src/App.css

```diff
@@ -1,38 +1,9 @@
 .App {
-  text-align: center;
+  background: var(--background-color);
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.app-grid {
+  display: grid;
+  grid-template-rows: [header] 160px [content] auto [footer] 80px;
+  height: 100vh;
```

Comment:
It's a bit strange, that you define the grid here, but then use the grid-rows in a different CSS file. You could also put them in here. Unless you want to set them up differently for every page :)